lib/console/deployments/observability/webhook/datadog.ex

```diff
@@ -0,0 +1,142 @@
+defmodule Console.Deployments.Observability.Webhook.Datadog do
+  @behaviour Console.Deployments.Observability.Webhook
+  import Console.Deployments.Observability.Webhook.Base
+
+  def associations(scope, payload, acc) do
+    handle_associations(scope, payload, acc)
+  end
+
+  defp handle_associations(:project, payload, acc) do
+    cond do
```

Comment:
I don't think this cond logic actually works, refactor it to use matches with multiple function heads instead (can be done for all association types, just imitate the existing grafana logic)